flashinfer/prefill.py

```
@@ -2040,26 +2040,80 @@ def plan(
-                    assert self._block_tables is not None, (
-                        "block_tables is not initialized"
+                # Try GPU preparation path: compute kv_lens, block_tables, and
+                # metadata entirely on the GPU, avoiding the Python for-loop and
+                # extra D2H/H2D transfers.
+                _fmha_v2_mod = None
+                try:
+                    _fmha_v2_mod = get_trtllm_fmha_v2_module(
+                        "q_paged_kv_hnd", q_data_type
+                    )
+                except Exception:
+                    pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)